03-cartography/exercises/turbo.md

```diff
@@ -0,0 +1,271 @@
+# TURBO VIEWER WORKSHOP
+
+## INDEX
+
+* [CARTOCSS vs. TURBO CARTO](#turbo)
+  * [Bubbles](#bubbles)
+  * [Categories](#categories)
+  * [Choropleth](#choropleth)
+  * [Color Brewer](#brewer)
+  * [Bivariate](#bivariate)
+  * [Building](#buildings)
+  * [Opacity](#opacity)
+  * [Lines](#lines)
+  * [Cartocolors](#cartocolors)
+* [RESOURCES](#resources)
+
+<hr>
+
+<a name="turbo"></a>
+### CARTOCSS vs. TURBO CARTO 
```

Comment:
This needs a bit of some introductory wording, what is the objective of this content? anything you should know beforehand? what will I learn? why?

Reply:
I added some wording explaining the purpose of this workshop.